beta/src/components/MDX/Challenges/Navigation.tsx

```diff
@@ -23,7 +23,9 @@ export function Navigation({
   const challengesNavRef = React.useRef(
     challenges.map(() => createRef<HTMLButtonElement>())
   );
-  const [scrollPos, setScrollPos] = React.useState(0);
+  const activeOrder =
+    challenges.find(({id}) => id === activeChallenge)?.order || 1;
```

Comment:
We have the `currentChallenge` in its parent component. Would it be good if we use that instead of re computing it again? 
https://github.com/reactjs/reactjs.org/blob/c6353b9853116680ce51352872a3df1078eedee0/beta/src/components/MDX/Challenges/Challenges.tsx#L99

Reply:
Yes, we could. I was using `activeChallenge` as it was already being passed through as a prop.

However, that could be replaced with `currentChallenge.name` in the Navigation function, then use the `order` attribute as I was already doing.

I will recommit, thank you for the suggestion.